src/jit/emit.h

```diff
@@ -764,7 +767,7 @@ class emitter
         // x86:   24 bits
         // amd64: 16 bits
         // arm:   14 bits
-        // arm64: 14 bits
+        // arm64: 12 bits
 
         unsigned _idSmallCns : ID_BIT_SMALL_CNS;
```

Comment:
We don't want the value of ID_BIT_SMALL_CNS to go below 12 bits.
It OK now, but we can't steal anymore bits from it on ARM64.